ledger/ledger.go

```diff
@@ -862,6 +862,14 @@ func (l *Ledger) Validate(ctx context.Context, blk bookkeeping.Block, executionP
 	return &vb, nil
 }
 
+// LatestTrackerCommitted returns the trackers' dbRound which "is always exactly accountsRound()"
+func (l *Ledger) LatestTrackerCommitted() basics.Round {
+	l.trackers.mu.RLock()
+	dbRound := l.trackers.dbRound
```

Comment:
`trackers.mu` should not be used outside of trackerRegistry. Please make a `trackerRegistry::DbRound()` method and call it there.

Reply:
I didn't see any other public methods on the `trackerRegistry` so I'm providing a private method `safeDbRound()`. LMK

Reply:
cf0b5ba52dbb812b63ab3ec485d3f2e83032c186